python/MooseDocs/common/build_class_database.py

```diff
@@ -57,11 +57,9 @@ def _locate_filenames(directories, ext):
 
     out = set()
     for location in directories:
-        for base, _, files in os.walk(location):
-            for fname in files:
-                full_file = os.path.join(base, fname)
-                if fname.endswith(ext) and not os.path.islink(full_file):
-                    out.add(full_file)
+        for filename in mooseutils.git_ls_files(os.path.join(MooseDocs.ROOT_DIR, location)):
```

Comment:
@andrsd This seems to be the problem right here.
This evaluates to `ROOT_DIR/${MOOSE_DIR}/framework/include` which `git ls-files` doesn't know about.
We haven't seen this in mastodon because the moose devel branch was broken before this PR went in, so it hasn't updated its submodule yet.